main.star

```diff
@@ -283,12 +285,17 @@ def run(plan, args={}):
             beacon_metrics_gazer_config_template = read_file(
                 static_files.BEACON_METRICS_GAZER_CONFIG_TEMPLATE_FILEPATH
             )
-            beacon_metrics_gazer.launch_beacon_metrics_gazer(
-                plan,
-                beacon_metrics_gazer_config_template,
-                all_cl_client_contexts,
-                args_with_right_defaults.participants,
-                network_params,
+            beacon_metrics_gazer_prometheus_metrics_job = (
+                beacon_metrics_gazer.launch_beacon_metrics_gazer(
+                    plan,
+                    beacon_metrics_gazer_config_template,
+                    all_cl_client_contexts,
+                    args_with_right_defaults.participants,
+                    network_params,
+                )
+            )
+            prometheus_additional_metrics_jobs.append(
```

Comment:
should we set `launch_prometheus_grafana` to true here as well? is this useful otherwise